packages/reactive_ble_mobile/protos/bledata.proto

```diff
@@ -70,6 +70,16 @@ message WriteCharacteristicInfo {
     GenericFailure failure = 3;
 }
 
+message WriteDescriptorRequest {
+    DescriptorAddress descriptor = 1;
+    bytes value = 2;
```

Comment:
@ened Do descriptors _always_ have 16-bit IDs and cannot have longer or shorter IDs?